provider/offline_test.go

```diff
@@ -375,21 +385,36 @@ func testMaterializations(t *testing.T, store OfflineStore) {
 		if err != nil {
 			t.Fatalf("Failed to create segment: %s", err)
 		}
+
 		i := 0
+		expectedRows := test.ExpectedSegment
 		for seg.Next() {
 			actual := seg.Value()
-			expected := test.ExpectedSegment[i]
-			if !reflect.DeepEqual(actual, expected) {
-				t.Fatalf("Value not equal in materialization: %v %v\n"+
-					"%T:%T %T:%T %T:%T (received:expected)\n", actual, expected, actual.Entity, expected.Entity, actual.Value, expected.Value, actual.TS, expected.TS)
+
+			found := false
+			for i, expRow := range expectedRows {
```

Comment:
So if things are ordering properly now, can we revert this back to how it was before?

Reply:
the materialization isn't necessarily ordered in the expected order.

Reply:
yes!

Reply:
Alright thats fine